apps/src/dance/ai/DanceAiModal.tsx

```diff
@@ -7,27 +7,28 @@ import {useAppDispatch} from '@cdo/apps/util/reduxHooks';
 import {closeAiModal, DanceState} from '../danceRedux';
 import classNames from 'classnames';
 import {FieldDropdown, Workspace} from 'blockly/core';
-import {
-  chooseEffects,
-  ChooseEffectsQuality,
-  getGeneratedEffectScores,
-} from './DanceAiClient';
 import DanceAiScore, {ScoreColors} from './DanceAiScore';
 import AiVisualizationPreview from './AiVisualizationPreview';
-import AiBlockPreview from './AiBlockPreview';
+
 import {
   AiFieldValue,
   AiOutput,
+  DanceAiModelItem,
   DanceAiSound,
   FieldKey,
   GeneratedEffect,
   MinMax,
+  ChooseEffectsQuality,
+  Mode,
+  Toggle,
```

Comment:
Both of these types are fast track candidates for renaming, but everything's wordy. 

`DanceAiModalMode`? `DanceAiPreviewButtonToggle`? I'm open to suggestions.

Reply:
Maybe `AiModalMode` and `AiPreviewButtonToggle`? I see we're using `AiOutput` already and omitting 'Dance' from it.

Reply:
Went with `DanceAiModalMode` and `DanceAiPreviewButtonToggleState`. Sound good?

Reply:
While we're here, what do you think of renaming `AiOutput` as `AiOutputType`? The current naming seems to imply it's the actual output

Reply:
moved it to `DanceAiModalOutputType`